dashboard/app/controllers/api/v1/json_api_controller.rb

```diff
@@ -5,4 +5,9 @@ class Api::V1::JsonApiController < ApplicationController
   rescue_from CanCan::AccessDenied do
     head :forbidden
   end
+
+  # Don't differentiate between missing and forbidden records
+  rescue_from ActiveRecord::RecordNotFound do
+    head :forbidden
+  end
```

Comment:
Not sure. What prompted this? 

It's probably fine. I don't think we're specifically reacting to 404s in the client differently from any other failure, but I'm not entirely sure. If we ever want a particular API to return not found instead we can always override that one.